autoarray/layout/region.py

```diff
@@ -91,6 +95,10 @@ def front_region_from(self, pixels: Tuple[int, int]) -> "Region1D":
             A tuple defining the pixel coordinates used to compute the front region.
         """
 
+        if pixels_from_end is not None:
+
+            pixels = (self.total_pixels - pixels_from_end, self.total_pixels)

```

Comment:
Maybe use a decorator to do this?

```python
def from_end(func):
    @functools.wraps(func)
    def wrapper(self, pixels=None, pixels_from_end=None):
        if pixels_from_end is not None:
            ...
```

That would save repetition and mean that these functions would still just take 'pixels' as an argument